src/components/UserAccess/UserAccessForm/UsernameSection.tsx

```diff
@@ -36,28 +35,25 @@ export const UsernameSection: React.FC<React.PropsWithChildren<Props>> = ({ disa
       setValidating(true);
       setValidHelpText('');
       setError('');
+      // username is mandantory
       if (!username) {
         setValidating(false);
         return;
       }
+      // usename format should be valid
       if (!usernameRegex.test(username)) {
         setValidating(false);
         setError('Invalid username format.');
         return;
       }
-      void validateUsername(username).then((valid) => {
-        setValidating(false);
-        if (valid) {
-          setError('');
-          setValidHelpText('Validated');
-          if (!usernames.includes(username)) {
-            void setValue([...usernames, username]);
-          }
-          setUsername('');
-        } else {
-          setError('Username not found.');
-        }
-      });
+      // for valid username
+      setValidating(false);
+      setError('');
+      setValidHelpText('Valid username format.');
+      if (!usernames.includes(username)) {
+        void setValue([...usernames, username]);
+      }
+      setUsername('');
```

Comment:
Remove validation for name, As for the RegEx validation, Only show failed state with a description of what is allowed for the username. Look at other validation in code
Something like this [RESOURCE_NAME_REGEX_MSG](https://github.com/konflux-ci/konflux-ui/blob/main/src/utils/validation-utils.ts#L7)